livestyled/models/product.py

```diff
@@ -144,12 +150,18 @@ def create_new(
 
     def diff(self, other):
         differences = {}
-        fields = (
-            'initial', 'on_hand', 'fulfilment_point', 'product_variant'
-        )
+        fields = ('initial', 'on_hand')
         for field in fields:
             if getattr(self, field) != getattr(other, field):
                 differences[field] = getattr(self, field)
+
+        related_resource_fields = ('fulfilment_point', 'product_variant')
```

Comment:
why?

Reply:
Same as above, seems like we used such an approach earlier too